packages/xdl/src/Project.ts

```diff
@@ -534,17 +535,24 @@ export async function exportForAppHosting(
     .update(iosBundle)
     .digest('hex');
   const iosBundleUrl = `ios-${iosBundleHash}.js`;
+  const iosBundleBytecodeUrl = `ios-${iosBundleHash}.hbc`;
```

Comment:
Please remove the lines for iOS from this PR. It doesn't make sense to have them here at this point.